### PR TITLE
Fix driver definition manifest: fsGroup is supported, snapshots are not

### PR DESCRIPTION
**Pull request by bertinatto**
CC @openshift/storage

### CODE REVIEW

File: test/e2e/manifest.yaml [hunk 1]
```diff
@@ -11,11 +11,11 @@ DriverInfo:
     exec: true
     multipods: true
     RWX: true
-    # TODO: Re-enable fsGroup tests: https://bugzilla.redhat.com/show_bug.cgi?id=2019128
+    # FSGroup is not supported in this release
     fsGroup: false
```
Comment (gnufied) [inline test/e2e/manifest.yaml:15]: Why is fsGroup not supported? I tested it last week for azurefile, it was working fine with mount options.

Comment (bertinatto) [reply test/e2e/manifest.yaml]: Fixed

Comment (bertinatto) [reply test/e2e/manifest.yaml]: From https://issues.redhat.com/browse/STOR-760:
```
The feature did not get GA in Kubernetes 1.24. It's beta there, enabled by default, and we will support it as GA for Azure File CSi driver.
```

File: test/e2e/manifest.yaml [hunk 2]
```diff
@@ -11,11 +11,10 @@ DriverInfo:
     exec: true
     multipods: true
     RWX: true
-    # TODO: Re-enable fsGroup tests: https://bugzilla.redhat.com/show_bug.cgi?id=2019128
-    fsGroup: false
+    fsGroup: true
```
[the remark this answers is not in the document]
Comment (bertinatto) [reply test/e2e/manifest.yaml:14]: That's probably correct, most fsgroup tests should skip on Azure File, and that decision relies on the `volumeMountGroup` cap [set to true]( https://github.com/bertinatto/kubernetes/blob/564b2049231c971b6e2e51c0822ecbf030da4094/test/e2e/storage/testsuites/fsgroupchangepolicy.go#L220-L221).

[the remark this answers is not in the document]
Comment (gnufied) [reply test/e2e/manifest.yaml:14]: yep that is okay I think. Those `fsGroupChangePolicy` tests are not applicable to azurefile anyways, but looks like we *are* running some fsGroup tests with both flags enabled, so we have some coverage at least.

**Pull request merged**